py/server/deephaven/parquet.py

```diff
@@ -56,19 +71,31 @@ def _build_parquet_instructions(col_instructions: List[ColumnInstruction] = None
     if max_dictionary_keys is not None:
         builder.setMaximumDictionaryKeys(max_dictionary_keys)
 
-    if is_legacy_parquet:
-        builder.setIsLegacyParquet(True)
+    if is_legacy_parquet is not None:
+        builder.setIsLegacyParquet(is_legacy_parquet)
+
+    if target_page_size is not None:
+        builder.setTargetPageSize(target_page_size)
+
+    if is_refreshing is not None:
+        builder.setIsRefreshing(is_refreshing)
 
     return builder.build()
 
 
-def read(path: str, col_instructions: List[ColumnInstruction] = None, is_legacy_parquet: bool = False) -> Table:
+def read(
+    path: str,
+    col_instructions: List[ColumnInstruction] = None,
+    is_legacy_parquet: bool = None,
+    is_refreshing: bool = None,
```

Comment:
For boolean type parameters I think it is best to have a default value, which won't leave the users to wonder if unset what they are.